pkg/tfgen/convert_test.go

```diff
@@ -74,7 +74,7 @@ func TestConvert(t *testing.T) {
 import * as pulumi from "@pulumi/pulumi";
 
 const config = new pulumi.Config();
-const regionNumber = config.get("regionNumber") || {
+const regionNumber = config.getObject("regionNumber") || {
```

Comment:
I think this must be because the tf11 code generator used get, but tf12 uses getObject. It's probably a safe change but I haven't dug into or thought too hard about it.